examples/get_pse_data_multiple_demo.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
I tested this notebook using [colab](https://github.com/ajdajd/fastquant.git). For some reason, the parallel version was 6-sec slower than the normal loop version. May be this is true only in colab.

Reply:
Thanks for trying it out @jpdeleon !

What date did you use? I run a notebook daily on Colab with `n_jobs=-1` and it speeds up my query by around half (Colab has 2 cores).